queue/bytes_queue.go

```diff
@@ -41,6 +41,20 @@ type queueError struct {
 	message string
 }
 
+func getUVarintSize(x uint64) int {
```

Comment:
Can you please add a docstring? I know what the function does but I'm not sure what it's for.

Reply:
Definitely  yes